package-lock.json

```diff
@@ -15,38 +15,6 @@
         "wattpm": "^2.35.1"
       }
     },
-    "../platformatic/packages/control": {
```

Comment:
dunno why we had this stuff in the `package-lock.json` (I used it temporarily, but apparently I stored it in the file). Anyway, this was causing the CI to fail. so I updated both here and on the other [PR](https://github.com/platformatic/watt-admin/pull/16)